tmt/steps/prepare/install.py

```diff
@@ -51,6 +51,10 @@ class PrepareInstall(tmt.steps.prepare.PreparePlugin):
     # Supported methods
     _methods = [tmt.steps.Method(name='install', doc=__doc__, order=50)]
 
+    # Supported keys
+    _keys = tmt.steps.prepare.PreparePlugin._keys + \
```

Comment:
I've just realised that this is not the greatest design. It requires the plugins to do something to have `on` applied. I will try to refactor this a bit and will see how it goes